source/code-snippets/indexes/searchIndexes.js

```diff
@@ -29,6 +29,24 @@ async function run() {
         await collection.createSearchIndex(index1);
         // end createSearchIndex example
 
+        // start vectorSearchIdx example
+        // Create a Vector Search index
+        const vectorSearchIdx = {
+            name: "vsidx1",
+            type: "vectorSearch",
+            definition: {
+                fields: [{
+                    type: "vector",
```

Comment:
I assume this is correct because Henry provided this code in the ticket description - but can we confirm that `type: 'vector'` is correct here?  It's strange that this is an example to create a `type: 'vectorSearch'` index but we also specify `type: 'vector'` in the description body.

Reply:
Based on the procedure [here under the Atlas CLI tab](https://www.mongodb.com/docs/atlas/atlas-vector-search/create-index/#procedure), it looks like this is the correct format for the JSON index model
```javascript
{
  "collectionName": "<name-of-collection>",
  "database": "<name-of-database>",
  "type": "vectorSearch",
  "name": "<index-name>",
  "fields":[
    {
      "type": "vector",
      "path": <field-to-index>,
      "numDimensions": <number-of-dimensions>,
      "similarity": "euclidean | cosine | dotProduct"
    },
    {
      "type": "filter",
      "path": "<field-to-index>"
    },
    ...
  ]
}
```